bcipy/gui/viewer/data_viewer.py

```diff
@@ -501,7 +501,7 @@ def main(data_file: str,
     parser.add_argument('-r',
                         '--refresh',
                         help='refresh rate in ms',
-                        default=500,
+                        default=300,
```

Comment:
This is the refresh rate of the viewer, not the EEG device. It was set to refresh the display every 500 ms (half second). Was this choppy? The downside to a more frequent refresh is more performance overhead.

Reply:
It wasn't buggy at 300 but I will revert since it is configurable!